packages/mermaid/src/diagrams/info/infoTypes.ts

```diff
@@ -0,0 +1,10 @@
+import type { DiagramDb } from '../../diagram-api/types.js';
+
+/**
+ * Info diagram DB.
+ */
+export interface InfoDb extends DiagramDb {
```

Comment:
nit: Casing.

```suggestion
export interface InfoDB extends DiagramDB {
```

Reply:
I took it from `DiagramDb` interface, and I thought of the same thing, `DB` should be in caps, but since it was like that I followed it, but I'll fix soon.

Reply:
Yes, following convention was the right call for normal PRs. 👍 

But, here, you're fixing the whole section, so let's change to `DiagramDB` too.

Reply:
Great! Should I rename files to `diagramDB.ts`? or `diagramDb.ts` is fine?

Reply:
Renaming files with only case changes is sadly a problem with git.
Different people having different git configs and OSes on handling casing leads to errors. 
Leaving the file name as-is shold be fine.

Reply:
Git marks files as [`Renamed`](https://github.com/mermaid-js/mermaid/pull/4486/files#diff-40265b49334bcbafb58c9046a0dfb71c5b05aafffcc19f3af4c32fc4c7c61661) if is has small partial difference, but not sure about the percentage exactly.

And since most DBs are in js, adding types and organize it will most likely make git mark those files as deleted and new files has been created as ts. So I'm not it would matter since most of them would be marked as deleted, but if you say so.